dbt/exceptions.py

```diff
@@ -421,3 +435,15 @@ def raise_duplicate_patch_name(name, patch_1, patch_2):
             patch_2,
         )
     )
+
+
+def raise_incorrect_version(path):
+    raise_compiler_error(
+        'The schema file at {} does not contain a version specifier. dbt '
+        'assumes that schema.yml files without version specifiers are version '
+        '1 schemas, but this file looks like a version 2 schema. If this is '
+        'the case, you can fix this error by adding `version: 2` to the top '
+        'of the file.\n\nOtherwise, please consult the documentation for more '
+        'information on schema.yml syntax: '
```

Comment:
can we just stick two newlines between the end of the sentence and the next line? It makes it easier to copy/paste